### DISCUSSION TITLE
I don't understand this instruction: Double tap to give the AI access to read your clipboard (I suggest holding all but the last key down then quickly double tapping the last key).

### DISCUSSION THREAD

**Discussion by bluevisor** [ILikeAI/AlwaysReddy#29] [General]
Can someone enlighten me please?

**Comment by ILikeAI** [comment]
Hey there, you're right that's pretty confusing. To give the AI access to read the contents of your clipboard you need to quickly double tap the record hotkey in under 0.5 seconds. So for example if your record hotkey is 'CTRL+ALT+R', I suggest you hold down 'CTRL+ALT' then while they are held down you quickly double tap 'R'.  Does this make sense?

I used to have a print statement that would print when the hotkey had successfully been double tapped, I may add that back in.

Let me know if that's still unclear 😊

**Reply by bluevisor** [reply]
ah, I get it now. thanks for the clarification!
